packages/bruno-electron/src/index.js

```diff
@@ -68,6 +68,7 @@ app.on('ready', async () => {
     mainWindow.maximize();
   }
 
+  mainWindow.show();
```

Comment:
You Should probably listen for the `ready` Event of the mainWindow here. The Ready Event is fired, when the content / side is loaded

Reply:
> You Should probably listen for the `ready` Event of the mainWindow here and then call mainWindow.show(). The Ready Event is fired, when the content / side is loaded

Yeah, I tried listening to the `ready-to-show` event, and I noticed that it was a proper approach as well. However, after implementing that method, the app's initial load time seems like increased (just visually). So, I decided to stick with the current implementation to mimic the faster load time. Thanks for the review; I'll update the PR.

Using `ready-to-show` event:

https://github.com/user-attachments/assets/db8f1d26-1541-41dd-b1c0-907669ad780c

Reply:
Agreed that `ready=to-show` is the technically correct solution.
The additional lag seems to be introduced by the time taken to render the first page

Approving this for now as it seems to solve for the distortion seen in windows